doc/userguide/rules/payload-keywords.rst

```diff
@@ -431,19 +424,29 @@ Format::
 +-----------------------+-----------------------------------------------------------------------+
 | [endian <type>]	| - big (Most significant byte at lowest address)			|
 |		       	| - little (Most significant byte at the highest address)		|
+|		       	| - dce (Allow the DCE module to determine the byte order)		|
 +-----------------------+-----------------------------------------------------------------------+
 | [string <num_type>]  	| 									|
 |		       	| - hex Converted data is represented in hex				|
 |		       	| - dec Converted data is represented in decimal			|
 |		       	| - oct Converted data is represented as octal				|
 +-----------------------+-----------------------------------------------------------------------+
-| [dce]			| Allow the DCE module determine the byte order				|
+| [dce]                 | Allow the DCE module to determine the byte order                      |
 +-----------------------+-----------------------------------------------------------------------+
-| [bitmask] <value>	| The AND operator will be applied to the extracted value		|
-|			| The result will be right shifted by the number of bits equal to the	|
-|			| number of trailing zeros in the mask					|
+| [bitmask] <value>     | The AND operator will be applied to the extracted value               |
+|                       | The result will be right shifted by the number of bits equal to the   |
```

Comment:
are we missing the text that was on line 444 here?

Reply:
Good catch.